HLTrigger/Configuration/python/customizeHLTforCMSSW.py

```diff
@@ -18,6 +17,16 @@
 #                     pset.minGoodStripCharge = cms.PSet(refToPSet_ = cms.string('HLTSiStripClusterChargeCutNone'))
 #     return process
 
+# Matching ECAL selective readout in particle flow, need a new input with online Selective Readout Flags
+def customiseFor17794(process):
+     for edproducer in producers_by_type(process, "PFRecHitProducer"):
+          if hasattr(edproducer,'producers'):
+               for pset in edproducer.producers:
+                    if (pset.name == 'PFEBRecHitCreator' or pset.name == 'PFEERecHitCreator'):
+                         if not hasattr(pset,'srFlags'):
+                              pset.srFlags = cms.InputTag('hltEcalDigis')
```

Comment:
it's not practical to have it in a producer because it doesn't know (and is not really supposed to know the configuration of possible plugins that it can call).
I was just stating the reasons why it's more practical to have this customise.